ui/pages/create-account/index.scss

```
@@ -87,6 +87,10 @@
     color: $scorpion;
     margin-top: 15px;
     padding: 0 20px;
+
+    &__error {
+      border: 1px solid $monzo;
```

This color is listed in `ui/css/design-system/deprecated-colors.scss`, which indicates that it generally should not be used. See the `ui/css/design-system/colors.scss` module for the set of colors we are supposed to be using going forward.

The most appropriate color here is probably `$error-3`

```suggestion
      border: 1px solid $error-3;
```

---

Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3